c/version.rc.in

```diff
@@ -0,0 +1,48 @@
+#include <windows.h>
+
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is it not possible to put the license header at the top?

Reply:
Added licence at the top